README.md

```diff
@@ -57,4 +96,45 @@ List of valid categories:
  15. Themes
  16. Other
 
-The official list of downloadable programs is kept on a public ReactOS server and synced every time RAPPS is launched for the first time.
+### URLSite
+Main web site where the program can be found. **__ Is HTTPS mandatory or only advised? If the link is in HTTP, can it try to switch automatically to HTTPS if avaliable? __**
+
+### URLDownload
+Direct link to the installer/program. **__ Is HTTPS mandatory or only advised? If the link is in HTTP, can it try to switch automatically to HTTPS if avaliable? __**
+
+### ScreenshotN
+Screenshot URL link. (N = 1, 2, 3 ...)
+
+Currently, only first Screenshot is used. Ensure the URL is a direct link to the image, not a page containing the image (e.g. Imgur links). It is also preferred to include the screenshot taken on ReactOS, not on Windows.  **__ Does it still support only one screenshot? Have you thought about creating a repository for screenshots? __**
```

Comment:
only one screenshot, yes.
We did think about creating something for that, but there are enough free image hosts (like imgur) that specialize in just this.
We are building an OS, we don't really need to also replicate every website and service out there.

Reply:
Ok, it was just to not have to depend on external services. Is there a way to document screenshot's copyright, apart from editing the image? (Something like Wikipedia, that has templates for screenshots of libre/copyrighted software etc.)